config/routes.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  
+  root to: "pages#root"
```

Comment:
All good with this for the test case.  If we are thinking of keeping it in place, can we change it to be a RESTful action/route (e.g. routing to `'welcome#index'`, or `'landing#index'`, or something to that effect)?

Reply:
@hpjaj any good ways to have a "catch-all" route in rails? Basically want every GET request that is not scooped up to just route to that controller to load our react app

Reply:
figured it out, you can do `get '*path'` where path **is not** a regex but instead a magic word

Reply:
Sweet.  Perhaps this could route to:

```
get '*path', to: 'pages#index'
```

as this route would be acting as the index source for all pages?

Reply:
updated and pushed